installer/tue-install.bash

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 _tue-check-env-vars || return 1
 
+[[ -n "$CI" ]] && set -o errexit
```

Comment:
This is wrong. The script breaks out because the branch of `tue-env` does not exist in `tue-env-targets`. The branch of `tue-env-targets` must be set to master in this case and `tue-install-impl` must be executed. Any errors in that script must break the CI.